portal-api/src/external-api/types/bank-holidays.d.ts

```diff
@@ -0,0 +1,31 @@
+export type BankHolidayRegion = 'england-and-wales' | 'scotland' | 'northern-ireland';
+
+type BankHolidayEvent = {
+  /*
+   * Title of the event - e.g. 'New Year's Day'
+   */
+  title: string;
+  /*
+   * Date of the event ISO 8601 format - e.g. '2022-01-03
```

Comment:
This is missing a closing quotation on the example date I think

Reply:
😢 

I'll fix in FN-1042